spec/models/room_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+describe "Room" do
+  it "must have a name" do
+    expect(Room.new).not_to be_valid
```

Comment:
This block isn't really testing that it `must have a name`, right? To finish that validation, I would add something like
```ruby
expect(Room.new(name: "room 1").to be_valid
```